xen/eg4/syn/project_vm2.tcl

```diff
@@ -0,0 +1,24 @@
+import_device eagle_s20.db -package BG256
```

Comment:
Comments:
1. please lowercase the title: Added -> add
2. is this vm2 specific? eg4 is the base project for all CPU? Why tcl file is named with "vm2"
3. please, provide a brief instruction how to use this tcl in commit message

Reply:
1. fixed
2. Because lib/config.h has `define CONFIG_CPU_VM2        1`, i.e. default target CPU is VM2.
3. this TCL script has been generated by Anlogic UI from corresponding XML file to build the project from command line.

Reply:
After the discussion we decided to elaborate this update for all CPUs
Postponed, closed